packages/commons-test/test/redisRateLimiter.integration.test.ts

```diff
@@ -0,0 +1,160 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { describe, expect, it } from "vitest";
+import { generateId, TenantId } from "pagopa-interop-models";
+import { genericLogger } from "pagopa-interop-commons";
+import { sleep } from "../src/testUtils.js";
+import { redisRateLimiter } from "./utils.js";
+
+describe("Redis rate limiter tests", async () => {
+  /*
+  ---------- NOTE ------------------------------------
+  Test rate limiter configuration defined in .env.test
+  ----------------------------------------------------
+  */
+
+  it("should rate limit requests by organizationId", async () => {
```

Comment:
in your opinion, does this test risk to be flacky?

Reply:
I'm confident it's not flaky! I also tried to run it tens of times locally and it always passes